runtime/tests/checker/account_test.go

```diff
@@ -764,11 +763,17 @@ func TestCheckAccount_borrow(t *testing.T) {
 					rType := RequireGlobalType(t, checker.Elaboration, "R")
 					rValueType := RequireGlobalValue(t, checker.Elaboration, "r")
 
+					xType := RequireGlobalType(t, checker.Elaboration, "X").(*sema.EntitlementType)
```

Comment:
Check type before casting with `require.IsType`, so test failure does not crash tests.

(For this test and the other tests in this file and other files)